pkgs/tools/virtualization/nixos-container/nixos-container.pl

```diff
@@ -34,12 +34,13 @@ sub showHelp {
          [--port <port>]
          [--host-address <string>]
          [--local-address <string>]
-       nixos-container destroy <container-name>
        nixos-container start <container-name>
-       nixos-container stop <container-name>
-       nixos-container terminate <container-name>
        nixos-container status <container-name>
        nixos-container update <container-name>
+       nixos-container restart <container-name>
+       nixos-container stop <container-name>  -- shutdown container cleanly, wait until stopped
+       nixos-container terminate <container-name> -- halt container, like hard poweroff
+       nixos-container destroy <container-name> -- terminate (halt) container and remove state data
```

Comment:
This should move into `nixos/doc/manual/administration/imperative-containers.xml` (if not already there). Why were things reordered?

Reply:
well, technically yes, but it's fine duplicating it here. No need to consult manual when stopping container.

Things were reordered to group stop/terminate/destroy commands. They do same, but with different nuances.